src/cli/README_DATASET.md

```diff
@@ -107,6 +107,108 @@ After the device successfully attaches to a Thread network, the device will retr
    Done
    ```
 
+### Using the Pending Operational Dataset for Delayed Dataset Updates
+
+The Pending Operational Dataset can be used for a delayed update of network parameters on all devices of a Thread Network.
+If certain Active Operational Dataset parameter(s) need to be changed, but the change would impact the connectivity of the network,
+delaying the update helps to let all devices receive the new parameter(s) before the update is applied. Examples of such
+parameters are the channel, PAN ID, certain Security Policy bits, or Network Key.
+
+The delay timer determines the time period after which the Pending Dataset takes effect and becomes the
```

Comment:
Please use the full "Pending Operational Dataset" and "Active Operational Dataset" throughout to align with other usage in the doc.